edge/pkg/edged/apis/types.go

```diff
@@ -2,7 +2,10 @@ package apis
 
 import (
 	"errors"
+	"github.com/docker/docker/api/types/container"
```

Comment:
This import should be in the last batch.

Reply:
done